src/main/topLevelPages/myDashboard/newVirtualContributorWizard/useVirtualContributorWizard.tsx

```diff
@@ -419,7 +430,15 @@ const useVirtualContributorWizard = (): useVirtualContributorWizardProvided => {
       return;
     }
 
-    setCreatedVc(createdVCData);
+    setCreatedVc({
```

Comment:
if you change that make avatar optional in that state you don't have to deconstruct here